ZackHolmberg/COMP4350-Project
# Issue 18: Initializing Transaction Service and Nginx Proxy

# Summary of Changes:

- Initialize a transaction flask service
- Setting up Docker files for the transaction service
- Initializing an Nginx Proxy to route the requests to the appropriate flask service
- Docker compose file to build the Transaction service and the nginx proxy

# How to test
- You will need docker installed on your computer
- Run `docker-compose build`, in the backend/services directory to build the docker containers
- Run `docker-compose up`, to spin up the containers
- localhost/transactions path on the web browser should reply 'Hello Transaction'

Closes #18
